examples/kitchen-sink/demo.tf

```diff
@@ -206,6 +206,18 @@ resource "sentry_metric_alert" "main" {
     threshold_type  = 0
   }
 
+  trigger {
+    action {
+      type              = "slack"
+      target_type       = "specific"
+      target_identifier = "#slack-channel"
+      integration_id    = 99999
```

Comment:
Perhaps it doesn't make much sense for us to provide a "demo" file that won't load (since 99999 doesn't correspond with an existing Sentry integration).